lib/TPP/DefaultPipeline.cpp

```diff
@@ -64,17 +64,19 @@ llvm::cl::opt<bool> lowerPackUnpackWithoutTranspose(
 
 // Lhs tile sizes for linalg-to-vector.
 llvm::cl::list<unsigned>
-    lhsTile("lhsTile",
-                     llvm::cl::desc("Lhs tile size for brgemm operation"),
-                     llvm::cl::list_init<unsigned>(SmallVector<unsigned>{8, 8}),
-                     llvm::cl::CommaSeparated);
+    lhsTile("lhsTile", llvm::cl::desc("Lhs tile size for brgemm operation"),
+            llvm::cl::list_init<unsigned>(SmallVector<unsigned>{8, 8}),
+            llvm::cl::CommaSeparated);
 
 // Rhs tile sizes for linalg-to-vector
 llvm::cl::list<unsigned>
-    rhsTile("rhsTile",
-                     llvm::cl::desc("Rhs tile size for brgemm operation"),
-                     llvm::cl::list_init<unsigned>(SmallVector<unsigned>{8, 16}),
-                     llvm::cl::CommaSeparated);
+    rhsTile("rhsTile", llvm::cl::desc("Rhs tile size for brgemm operation"),
+            llvm::cl::list_init<unsigned>(SmallVector<unsigned>{8, 16}),
+            llvm::cl::CommaSeparated);
+
+llvm::cl::opt<bool> vectorToXSMM("vector-To-XSMM",
```

Comment:
I tried to use the same flag as the one in passbundles, I get an error saying there's no such option in tpp-run. However, if I try to add another command with the same name, it complains that the command already exists. I didn't know to resolve it immediately and added a command with a different name for now.

Reply:
Please don't just create a new option if you can't use an existing one.

For `tpp-opt`, because you have to pass `-default-tpp-passes`, you need to add the flag to that one, for example `-default-tpp-passes="vector-to-xsmm"`. In `tpp-run`, because the default TPP passes will run by default, you can just call `-vector-to-xsmm`.

Reply:
I tried invoking tpp-run with --vector-to-xsmm. I get an error "Unknown command line argument". Can you please let me know how to pass vector-to-xsmm flag to tpp-run? I don't see it in help either.

Reply:
Can you please tell me how to pass the right argument? I cant run tests and I have no clue how to pass the argument, I tried everything, all variants of vector-to-xsmm, nothing seems to work.

Reply:
I had to add the argument to tpp-run.cpp, just like gpuBackend commandline argument, and that is working.